scripts/ScriptUtilities.psm1

```diff
@@ -61,15 +61,10 @@ function New-NuGetPackageFromNuspecFile($configuration, $project, $version, $fra
 }
 
 function New-NuGetPackages($configuration, $projects, $frameworks) {
-    $versionPrefix, $versionSuffix = & $PSScriptRoot\Get-VersionConstants.ps1
```

Comment:
>Get-VersionConstants.ps1 [](start = 53, length = 24)

The file Get-VersionConstants.ps1 should no longer be needed and can be removed. (If it _is_ still needed, then we missed something.)